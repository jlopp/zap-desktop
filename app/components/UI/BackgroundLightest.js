import React from 'react'
import { Box } from 'rebass'

/**
 * @render react
 * @name BackgroundLightest
 * @example
 * <BackgroundLightest />
 */
class BackgroundLightest extends React.Component {
  render() {
    return <Box bg="lightestBackground" color="white" {...this.props} />
  }
}

export default BackgroundLightest
