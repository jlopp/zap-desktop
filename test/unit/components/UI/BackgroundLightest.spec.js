import React from 'react'
import BackgroundLightest from 'components/UI/BackgroundLightest'
import renderer from 'react-test-renderer'

describe('component.UI.BackgroundLightest', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<BackgroundLightest />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
