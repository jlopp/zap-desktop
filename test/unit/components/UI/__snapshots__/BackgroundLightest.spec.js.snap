// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.BackgroundLightest should render correctly 1`] = `
.c0 {
  color: white;
  background-color: lightestBackground;
}

<div
  className="c0"
  color="white"
/>
`;
